WordPress/Classes/Plugins/Views/AddNewPluginView.swift

```diff
@@ -42,6 +42,17 @@ struct AddNewPluginView: View {
         .task(id: 0) {
             await viewModel.onAppear()
         }
+        .task(id: 1) {
```

Comment:
The `id` is used to cancel and re-create tasks, so using a constant value doesn't do anything, and it can be removed.

```swift
This method behaves like [task(priority:_:)](https://developer.apple.com/documentation/swiftui/view/task(priority:_:)), except that it also cancels and recreates the task when a specified value changes. To detect a change, the modifier tests whether a new value for the id parameter equals the previous value. For this to work, the value’s type must conform to the [Equatable](https://developer.apple.com/documentation/Swift/Equatable) protocol.
```

Reply:
I thought `.task { ... }` would get called when on view updates, but giving it a constant `.task(id: 1) { ... }` could prevent the async block from being called when view updates.

Reply:
I can't reproduce ⬆️ now. I'm certain I have seen repeated calls...

Anyway, I'll remove those constants in https://github.com/wordpress-mobile/WordPress-iOS/pull/24092/commits/0120ef76ea53952c4e41031fd817713c4de4f986

Reply:
>  would get called when on view updates

It wouldn't be possible because `body` can potentially be called thousands of times depending on how frequent the updates are. It is [documented](https://developer.apple.com/documentation/swiftui/view/task(priority:_:)) to run before view appears and be cancelled when it disappears.